common/thorhelper/thorfile.cpp

```diff
@@ -86,6 +86,16 @@ class DiskWorkUnitReadArg : public CThorDiskReadArg
         memcpy(rowBuilder.ensureCapacity(size, NULL), src, size);
         return size;
     }
+    virtual const char * queryFormat() override
+    {
+        return "thor";
```

Comment:
Is this preferred to "flat" ?

Reply:
no.  It is one of the things I plan to change.